package-lock.json

```diff
@@ -5,7 +5,8 @@
   "requires": true,
   "packages": {
     "": {
-      "version": "0.2.0",
+      "name": "eslint-plugin-react-hook-form",
+      "version": "0.2.4",
```

Comment:
Is there any reason that we need to add this? I suppose package-lock.json should be modified by the npm.

Reply:
It's probably not necessary. I assume this was added by npm when I ran `npm install` and I didn't notice it. I will remove it.